frontend/app/src/components/helper/Card.vue

```diff
@@ -46,39 +44,26 @@ const otherHeights = computed(() => {
     v-bind="rootAttrs"
     :flat="flat"
     :class="{
-      [css['no-radius-bottom']]: noRadiusBottom,
-      [css['full-height']]: fullHeight
+      ['h-full']: fullHeight
     }"
+    :outlined="outlined"
     v-on="rootListeners"
   >
-    <VCardTitle v-if="slots.title" :class="{ 'pt-6': slots.icon }">
-      <slot v-if="slots.icon" name="icon" />
-      <CardTitle
-        :class="{
-          'ps-3': slots.icon,
-          [css.title]: slots.icon
-        }"
-      >
+    <VCardTitle v-if="slots.title">
+      <CardTitle>
         <slot name="title" />
       </CardTitle>
       <VSpacer v-if="slots.details" />
       <slot name="details" />
     </VCardTitle>
-    <VCardSubtitle v-if="slots.subtitle" :class="{ 'ms-14': slots.icon }">
-      <div
-        :class="{
-          'pt-2': slots.icon,
-          [css.subtitle]: slots.icon
-        }"
-      >
-        <slot name="subtitle" />
-      </div>
+    <VCardSubtitle v-if="slots.subtitle">
+      <slot name="subtitle" />
     </VCardSubtitle>
     <VCardText
       ref="bodyRef"
       :class="{
         [css.contained]: contained,
-        [css['no-padding']]: noPadding
+        ['no-padding']: noPadding
       }"
     >
       <slot name="search" />
```

Comment:
Can't select the actual place but can we replace the VSheet with a normal div for actions? from what I can see in the new designs there is no outline and we need to move as much as possible away from Vuetify either way